docs/diagnostics/PX1026.md

```diff
@@ -0,0 +1,45 @@
+# PX1026
+This document describes the PX1026 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                               | Type  | Code Fix  | 
+| ------ | --------------------------------------------------------------- | ----- | --------- | 
+| PX1026 | Underscores cannot be used in the names of DACs and DAC fields. | Error | Available |
+
+## Diagnostic Description
+Underscores cannot be used in the names of DACs and DAC fields. 
```

Comment:
@k-s-popova these applies only to the public and internal DAC properties and corresponding nested classes. If there are no DAC property coresponding to the nested class then the diagnostic doesn't apply to it.

Reply:
@k-s-popova It's also worth mentioning the reason behind this diagnostic - reference to the DAC property in aspx would be interpreted incorrectly if it contains underscores since it is  a special symbol for aspx

Reply:
@SENya1990, both fixed.

Reply:
@k-s-popova, looks fine, thanks